packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -232,5 +232,11 @@ module.exports = {
     fs: 'empty',
     net: 'empty',
     tls: 'empty'
+  },
+  // Turn off performance hints during development because we don't do any
+  // splitting or minification in interest of speed. These warnings become
+  // cumbersome.
+  performance: {
+    hints: false
```

Comment:
What kind of additional details are you looking for? At the least I'd recommend having it as "warning" in Production as well.